src/lib/navigation/navigate.ts

```diff
@@ -19,9 +19,16 @@ export interface ViewDescriptor extends ViewOptions {
   props: object
 }
 
+export interface BackProps {
+  fromScreen?: string
+}
```

Comment:
I agree with George's comment below about the naming
```suggestion
export interface GoBackProps {
  previousScreen?: string
}
```

Reply:
Yes, otherwise there will be such a type incompatibility error
![image](https://user-images.githubusercontent.com/3513494/150963516-d5f23bae-e9b6-4377-9425-57edba998412.png)

Reply:
Got it. I'll change it